node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Looks like this might be the cause of adding the ID NPM library. I would probably remove it from this pull request because it isn't mandatory and is probably just a metadata file. Let me know your thoughts 👍